### ISSUE TITLE
proxy connection lost

### ISSUE THREAD

**Issue by piyush94** [DeployHubProject/DeployHub-Pro#333]
When deployhub loses connection with the reverse proxy, and a deployment is initiated, it returns deployment id as -1 but success field is returned as true. Success should be returned as false.
```
Tue Feb 23 09:45:29 2021: DOING deployment log length=2
Tue Feb 23 09:45:29 2021: {"iscomplete":false,"success":true}
Tue Feb 23 09:45:33 2021: Engine has ended!
Tue Feb 23 09:45:33 2021: Got it -1
Tue Feb 23 09:45:33 2021: {"success":true,"deploymentid":-1}
```

**Comment by sbtaylor15** [comment]
@piyush94 which logs are you seeing this in?  On the proxy side or on the server side?  Also, are you making an API call and getting back the incorrect data?

**Comment by piyush94** [comment]
@sbtaylor15 The logs are from the server-side DeployHub container. Getting wrong data in API call as well.

**Comment by sbtaylor15** [comment]
@piyush94 updated the json as follows when the reverse proxy is offline.

`{"deploymentid":-1,"success":false,"error":"Timed out waiting for deployment to start"}`

Fix is in `ui-skin-v9.0.0.3138-g3af5090`

Let us know if you need the fix in the old UI.
